sdk/servicebus/Azure.Messaging.ServiceBus/src/Client/ServiceBusClientOptions.cs

```diff
@@ -52,6 +52,13 @@ public ServiceBusRetryOptions RetryOptions
             }
         }
 
+        /// <summary>
+        /// If set to true, transactions spanning multiple entities can be performed.
+        /// This should not be set to true if not using transactions, or if all transactions
+        /// will be limited to a single entity.
+        /// </summary>
```

Comment:
```suggestion
        /// <summary>
        /// Gets or sets a flag that indicates whether or not transactions may span multiple
        /// Service Bus entities.  
        /// </summary>
        ///<value>
        /// <c>true</c>, when cross-entity transactions are enabled; <c>false</c> when
        /// transactions are not being used or should be limited to a single entity.
        ///</value>
```

I tried to remove the double-negative and separate out the intent of the property from how it's used.